src/wp-admin/includes/class-wp-automatic-updater.php

```diff
@@ -854,6 +862,190 @@ protected function send_email( $type, $core_update, $result = null ) {
+			return;
+		}
+		$body = array();
+
+		switch ( $type ) {
+			case 'success':
+				/* translators: %s: Site title. */
+				$subject = __( '[%s] Some plugins or themes were automatically updated' );
+				break;
+			case 'fail':
+				/* translators: %s: Site title. */
+				$subject = __( '[%s] Some plugins or themes have failed to update' );
+				$body[]  = sprintf(
+					/* translators: %s: Home URL. */
+					__( 'Howdy! Failures occurred when attempting to update plugins/themes on your site at %s.' ),
+					home_url()
+				);
+				$body[] = "\n";
+				$body[] = __( 'Please check out your site now. It’s possible that everything is working. If it says you need to update, you should do so.' );
+				break;
+			case 'mixed':
+				/* translators: %s: Site title. */
+				$subject = __( '[%s] Some plugins or themes were automatically updated' );
+				$body[]  = sprintf(
+					/* translators: %s: Home URL. */
+					__( 'Howdy! Failures occurred when attempting to update plugins/themes on your site at %s.' ),
+					home_url()
+				);
+				$body[] = "\n";
+				$body[] = __( 'Please check out your site now. It’s possible that everything is working. If it says you need to update, you should do so.' );
+				$body[] = "\n";
+				break;
+		}
+
+		// Get failed plugin updates.
+		if ( in_array( $type, array( 'fail', 'mixed' ), true ) && ! empty( $failed_updates['plugin'] ) ) {
+			$body[] = __( 'The following plugins failed to update:' );
+			// List failed updates.
+			foreach ( $failed_updates['plugin'] as $item ) {
+				/* translators: %s: Name of the related plugin. */
+				$body[] = ' ' . sprintf( __( '- %s' ), $item->name );
+			}
+			$body[] = "\n";
+		}
+		// Get failed theme updates.
+		if ( in_array( $type, array( 'fail', 'mixed' ), true ) && ! empty( $failed_updates['theme'] ) ) {
+			$body[] = __( 'The following themes failed to update:' );
+			// List failed updates.
+			foreach ( $failed_updates['theme'] as $item ) {
+				/* translators: %s: Name of the related plugin. */
+				$body[] = ' ' . sprintf( __( '- %s' ), $item->name );
+			}
+			$body[] = "\n";
+		}
+		// Get successful plugin updates.
+		if ( in_array( $type, array( 'success', 'mixed' ), true ) && ! empty( $successful_updates['plugin'] ) ) {
+			$body[] = __( 'The following plugins were successfully updated:' );
+			// List successful updates.
+			foreach ( $successful_updates['plugin'] as $item ) {
+				/* translators: %s: Name of the related plugin. */
+				$body[] = ' ' . sprintf( __( '- %s' ), $item->name );
+			}
+			$body[] = "\n";
+		}
+		// Get successful theme updates.
+		if ( in_array( $type, array( 'success', 'mixed' ), true ) && ! empty( $successful_updates['theme'] ) ) {
+			$body[] = __( 'The following themes were successfully updated:' );
+			// List successful updates.
+			foreach ( $successful_updates['theme'] as $item ) {
+				/* translators: %s: Name of the related plugin. */
+				$body[] = ' ' . sprintf( __( '- %s' ), $item->name );
```

Comment:
This line can be replaced with:
`$body[] = "- {$item->name}";`

For reference: https://github.com/WordPress/wp-autoupdates/pull/138/files

Reply:
Thanks for the update 👍